bin/metapolator-export

```diff
@@ -47,7 +47,16 @@ if (require.main === module) {
                 master = masterInstancePairs.shift();
                 instance = masterInstancePairs.shift();
                 console.warn('Exporting "'+master+'" to "'+instance+'"');
-                project.exportInstance(master, instance, program.precision);
+
+                String.prototype.endsWith = function(suffix) {
```

Comment:
We don't do ad-hoc patching of built-in prototypes. It's considered bad style in JavaScript. There's also no explicit code to do upgrading of missing functions that will be in future versions of JS (polyfills).
We could think about a way of adding the Polyfills defined at mdn (like: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith#Polyfill) but we don't have it yet. For some more important API's we do use something like polyfills or similar things i.e. Generators or Proxies

I would maybe do inline `if(instance.slice(-4) === '.zip')` as this is still comprehensible or define endsWith like `function endsWith(string, suffix)`